docs/detections/alerts-reduce.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[reduce-notifications-alerts]]
+== Reduce notifications and alerts
+
+{elastic-sec} offers several features to help reduce the amount of notifications and alerts produced by your detection rules. This table provides a general comparison of these features, with links for more details:
+
+[cols="2"]
+|===
+
+| <<snooze-rule-actions,Rule action snoozing>>
+a| *_Prevents a specific rule's notification actions from running_*. 
+
+Use to avoid unnecessary notifications from a noisy rule. The rule continues to run and generate alerts during the snooze period, but its <<rule-notifications,notification actions>> are suppressed.
+
+| {kibana-ref}/maintenance-windows.html[Maintenance window]
+a| *_Prevents all rules' notification actions from running_*. 
+
+Use to avoid false alarms and unnecessary notifications during planned outages. All rules continue to run and generate alerts during the maintenance window, but their <<rule-notifications,notification actions>> are suppressed.
```

Comment:
Making this suggestion in case we want to avoid over-using "suppression".

```suggestion
Use to avoid false alarms and unnecessary notifications during planned outages. All rules continue to run and generate alerts during the maintenance window, but their <<rule-notifications,notification actions>> are stopped.
```